react/components/SearchBar/components/SearchBar.js

```diff
@@ -5,10 +5,12 @@ import AutocompleteInput from './AutocompleteInput'
 import ResultsLists from './ResultsList'
 import DownshiftComponent from 'downshift'
 import { NoSSR } from 'vtex.render-runtime'
+import { Overlay } from 'vtex.react-portal'
 
 import styles from '../styles.css'
 
 export default class SearchBar extends Component {
+  container = React.createRef()
```

Comment:
Would you mind porting that to a function component?

Reply:
@klzns I will leave that to another PR